Tasks/VsTestPlatformToolInstallerV1/Tests/TestSetup.ts

```diff
@@ -19,9 +19,9 @@ const nugetToolPath = path.join(__dirname, '..', 'nuget.exe');
 const downloadPath = process.env[constants.downloadPath];
 
 // Construct commands to be mocked
-const listPackagesCommand = `${nugetToolPath} list packageid:${constants.packageId} ${(process.env[constants.versionSelector] === 'latestPreRelease' ? '-PreRelease ' : '')}-Source ${constants.defaultPackageSource}`;
```

Comment:
does this take care when path contains spaces? 🍏

Reply:
this is test code. 

The way we actually add in the prod code is using tool.arg which takes care